### PR TITLE
11.0 add hr associate

### PR DESCRIPTION
**Pull request by etobella**

### CODE REVIEW

File: cb_hr_views/models/hr_employee.py [hunk 1]
```diff
@@ -222,8 +222,15 @@ def action_open_related_partner(self):
 
     @api.multi
     def _compute_show_leaves(self):
-        for employee in self:
-            employee.show_leaves = employee.show_info
+        for rec in self:
+            rec.show_leaves = rec.show_info and rec.is_employee
```
Comment (jarroyomorales) [inline cb_hr_views/models/hr_employee.py:227]: Deberiamos añadir show_info y is_employee al api.depends()?

File: cb_hr_views/models/hr_employee.py [hunk 2]
```diff
@@ -222,8 +222,15 @@ def action_open_related_partner(self):
...
+
+    @api.multi
+    def _compute_can_see_examinations_button(self):
+        super()._compute_can_see_examinations_button()
+        for record in self:
+            if not record.is_employee:
+                record.can_see_examinations_button = False
```
Comment (jarroyomorales) [inline cb_hr_views/models/hr_employee.py:235]: Same

File: cb_hr_views/models/hr_employee.py (hunk 1 above)
Comment (etobella) [reply cb_hr_views/models/hr_employee.py:227]: ufff, well, as show_info is computed all the relateds of show_info, isn't it?

Comment (jarroyomorales) [reply cb_hr_views/models/hr_employee.py:227]: well, if it works this way lets keep it like this

**Pull request closed without merging**